rust/sdk-processor/src/config/indexer_processor_config.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use super::processor_config::ProcessorConfig;
+use crate::processors::events_processor::EventsProcessor;
+use anyhow::Result;
+use aptos_indexer_processor_sdk::aptos_indexer_transaction_stream::TransactionStreamConfig;
+use sdk_server_framework::RunnableConfig;
+use serde::{Deserialize, Serialize};
+
+pub const QUERY_DEFAULT_RETRIES: u32 = 5;
```

Comment:
are these constants used?

Reply:
They will be used once we migrate more processors, I can remove them for now